python/lsst/cp/pipe/defects.py

```diff
@@ -74,16 +77,39 @@ class MeasureDefectsTaskConfig(pipeBase.PipelineTaskConfig,
     """Configuration for measuring defects from a list of exposures
     """
 
+    thresholdType = pexConfig.ChoiceField(
+        dtype=str,
+        doc=("Defects threshold type: 'STDEV' or `VALUE`. If 'VALUE', cold pixels will be found "
+             "in flats, and hot pixels in darks. If 'STDEV', cold and hot pixels will be found "
+             "in flats, and hot pixels in darks."),
+        default='STDEV',
+        allowed={'STDEV': "Use number of sigma given standard deviation.",
+                 'VALUE': "Use pixel value."},
```

Comment:
Maybe these descriptions should say "for threshold" (but that may be redundant in context).  It just seems a bit terse as it is.

Reply:
Yeah, something like "Use pixel value to determine detection threshold" or "Use image standard deviation to determine detection threshold" or something like that.